python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py

```diff
@@ -32,17 +32,27 @@
 
 
 @dataclass
-class StaticTableDetails:
+class StaticTableColumns:
     """
-    Container class for loading static gtfs schedule tables
+    Container Class to hold conversion information for gtfs schedule tables
     """
 
-    table_name: str
-    insert_table: sa.sql.schema.Table
     columns_to_pull: List[str]
     int64_cols: Optional[List[str]] = None
     bool_cols: Optional[List[str]] = None
     time_to_seconds_cols: Optional[List[str]] = None
+
+
+@dataclass
+class StaticTableDetails:
+    """
+    Container class for loading static gtfs schedule tables
+    """
+
+    table_name: str
+    insert_table: sa.sql.schema.Table
+    static_version_key_column: sa.sql.schema.Column
+    column_info: StaticTableColumns
     data_table: pandas.DataFrame = pandas.DataFrame()
```

Comment:
i added the `static_version_key_column` that we do the rollback delete `where` logic on.

that triggered [R0902](https://pylint.readthedocs.io/en/stable/user_guide/messages/refactor/too-many-instance-attributes.html) errors in pylint, requiring the nesting of column info.